app/src/main/java/com/jerboa/ui/components/common/MarkdownHelper.kt

```diff
@@ -141,14 +142,23 @@ object MarkdownHelper {
             // use TableAwareLinkMovementMethod to handle clicks inside tables,
```

Comment:
This should only be part of the main markdown. I assume this would make it possible if a table with links is in the preview and you click on it it wont open the post but this link directly. Same goes for normal links. It should not allow you to open them.